src/main/java/org/gridsuite/study/server/repository/PublicStudy.java

```diff
@@ -0,0 +1,52 @@
+package org.gridsuite.study.server.repository;
```

Comment:
I think we can share the StudyEntity between the 3 repositories ?

Reply:
wa can't the name of the class references the name of the table